addons/killtracker/initSettings.inc.sqf

```diff
@@ -0,0 +1,10 @@
+[
+    QGVAR(trackAI),
+    "CHECKBOX",
+    [LSTRING(TrackAI_DisplayName), LSTRING(TrackAI_Description)],
+    LSTRING(Category),
+    true,
+    true,
```

Comment:
```suggestion
    1,
```

Reply:
Is there any meaningful difference?
Or just the way that ACE does this, makes it easier to see?

Reply:
https://github.com/CBATeam/CBA_A3/wiki/CBA-Settings-System#arguments-of-cba_fnc_addsetting